index.html

```diff
@@ -301,6 +301,55 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+A series of at least the following registries that support the normative
+deliverables of the Working Group will be published and maintained:
+          </p>
```

Comment:
We have to be careful with the fact that the term 'registry' is somewhat overloaded among W3C groups. I suspect the goal is to define formal registries in the W3C sense; if so, I think this should be made clear in the text. Something like

> A series of at least the following registries that support the normative deliverables of the Working Group will be published and maintained under the [W3C Registry Track](https://www.w3.org/2021/Process-20211102/#registries):

I leave to @msporny to find the best formulation. (If the comments of @Sakurann are accepted, those should be modified accordingly, too.)

Reply:
Addressed in c47ffd50ad6224d41e6bc7deabb627aef7a78db2.